src/app.ts

```diff
@@ -1,19 +1,16 @@
-import express, { Application } from 'express'
-import bodyParser from 'body-parser'
-import connect from "./connect"
-import * as ProductController from "./controllers/product_controller"
-import * as CategoryController from "./controllers/category_controller"
-import * as LocationController from "./controllers/location_controller"
-import * as CustomerController from "./controllers/customer_controller"
-import * as RevenueController from "./controllers/revenue_controller"
-import * as OrderController from "./controllers/order_controller"
-import * as OrderDetailController from "./controllers/orderDetail_controller"
-import * as StockController from "./controllers/stock_controller"
-import * as SupplierController from "./controllers/supplier_controller"
-import { asyncMiddleware } from './middlewares/async'
-import cors from 'cors'
-
-
+import express, { Application } from "express";
+import bodyParser from "body-parser";
+import connect from "./connect";
+import cors from "cors";
+import productRouter from "./routes/productRoute";
+import categoryRouter from "./routes/categoryRoute";
+import locationRouter from "./routes/locationRouter";
+import customerRouter from "./routes/customerRoute";
+import revenueRouter from "./routes/revenueRoute";
+import orderRouter from "./routes/orderRouter";
+import orderDetailRouter from "./routes/orderDetailRoute";
+import stockRouter from "./routes/stockRoute";
+import supplierRouter from "./routes/supplierRoute";
 
 const app: Application = express();
 const port = 3001;
```

Comment:
Look for a way to add the port to env variables.
https://stackoverflow.com/questions/18864677/what-is-process-env-port-in-node-js